packages/ckeditor5-link/docs/features/link.md

```diff
@@ -240,15 +240,17 @@ ClassicEditor
 
```

Comment:
I think this feature is important enough (and also far away in the guide) to mention it in the guide introduction (the first section of the guide). Otherwise people are unlikely to discover it.

This mention should also include the fact that the feature is not available by default so something like "optional autolink feature" would make sense here.